### DISCUSSION TITLE
former2 and cli have not delivered any results for several days

### DISCUSSION THREAD

**Discussion by dietmar-mueller** [iann0036/former2#380] [General]
i have used the former2 website as well as the cli options so far without any problems and with a successful scan and result.

for a few days now the scan has been hanging and no service data has been delivered and as a result no data can be generated.

the web frontend hangs on the last two services and terminates without a message.

with the developer tools of the browser no network traffik is logged. so i used burp as a proxy and analyzed the traffic.

it behaves in such a way that i get several option requests against the endpoint https://codestar.us-east-1.amazonaws.com without any result. i do not see any status code in burp. i suspect that it is a long-term timeout, but it seems that the request is stuck.

i have tried the scan via several different internet providers to rule out a local network problem. i have been seeing this behavior constantly for 5 days now.

how does this app generally behave with https connet problems. is there a hard timeout and does the code then continue to run?

in my case, this scan takes many minutes and then no scan result is returned.

thanks for your constructive feedback

ps: according to the status page of aws there is no problem with this service

![former2](https://github.com/user-attachments/assets/98e63c26-865f-47b9-9778-c37325b8d43f)

**Comment by iann0036** [comment]
Hi @dietmar-mueller,

It looks like now that CodeStar has been formally deprecated, the AWS service teams are doing...weird things to the endpoints.

I've removed the logic and links to the CodeStar page from Former2: https://github.com/iann0036/former2/commit/d509eea217f5c5413f75210e327684e4d5a30333

Let me know if that resolves your issue.

**Comment by dietmar-mueller** [comment]
hello @iann0036 ,

many thanks for the quick fix.

i can confirm that both: the webfronten https://former2.com/ and the cli are delivering the desired results again!

Thanks!

i have analyzed the whole thing again in more detail and saw that an unknown host was logged several times. however, this does not cause any problems with my service situation at aws.

this is just for your information.

these endpoints provide an unknonw host:

networkmanager.us-east-1.amazonaws.com
iotthingsgraph.us-east-1.amazonaws.com
worklink.us-east-1.amazonaws.com
devicefarm.us-east-1.amazonaws.com
devices.iot1click.us-east-1.amazonaws.com

best regards dietmar
